src/scenes/Preloader.ts

```diff
@@ -33,6 +35,26 @@ export class Preloader extends Scene
         this.load.setPath('assets');
 
         this.load.image('logo', 'logo.png');
+
+        this.load.image('play-icon', 'play-icon.png');
+        
+        // Test Only- replace with actual duck spritesheet
+        this.load.image('duck', 'duck.png');
+
+        // Load custom font
+        this.load.font(
+        'PixelGame',
+        'https://raw.githubusercontent.com/google/fonts/refs/heads/main/ofl/pressstart2p/PressStart2P-Regular.ttf',
```

Comment:
So normally i'd recommend avoiding using files from other repos. You don't really get to control if they change it or not. Can lead to security issues.

The repo being owned by google its proabably fine. I would replace main with the git sha, so if someone does delete the files or replace it, it still works.

So using github web ui, you goto https://github.com/google/fonts/blob/main/ofl/pressstart2p/PressStart2P-Regular.ttf
press "Y" key
that changes the url to `https://github.com/google/fonts/blob/7d1c962b6fbc8987b59ed229bbe5c12764d3a624/ofl/pressstart2p/PressStart2P-Regular.ttf`
so
```suggestion
        'https://raw.githubusercontent.com/google/fonts/refs/heads/7d1c962b6fbc8987b59ed229bbe5c12764d3a624/ofl/pressstart2p/PressStart2P-Regular.ttf',
```

(also recommend indenting it)

Reply:
New hack! I tried it but got a CORS error so I am using a local font instead. Also fixed the indentation. Appreciate it